hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -760,7 +761,7 @@ public static String normalizeKey(String keyName,
     if (!StringUtils.isBlank(keyName)) {
       String normalizedKeyName;
       if (keyName.startsWith(OM_KEY_PREFIX)) {
-        normalizedKeyName = new Path(keyName).toUri().getPath();
+        normalizedKeyName = new Path(normalizeDoubleSlashPath(keyName)).toUri().getPath();
       } else {
         normalizedKeyName = new Path(OM_KEY_PREFIX + keyName)
```

Comment:
I may be missing something, but the existing logic and addition of `normalizeDoubleSlashPath` seems unnecessarily complex to me.  Double `/` is handled in `normalizeDoubleSlashPath`, 1 or 3+ `/` by `Path`, no slash by `else`.  Also, for double slash we create `Path` twice (once new function, once here).

Can we just remove prefix of any number of `/`, add a single `/`, and pass that through `new Path(...).toUri().getPath()`?

Reply:
Ok, initially I just tried to remove the extra / and leave only one /, but in order to reduce the number of possible for loops, I took this fixed removal approach, but it seems that this is unnecessary